octavia_f5/tests/unit/controller/worker/tasks/test_f5_tasks.py

```diff
@@ -199,7 +199,7 @@ def test_SyncSubnetRoutes(self, mock_get_subnet):
             ]
         )
 
-        engines.run(f5_tasks.SyncSubnetRoutes(),
+        engines.run(f5_tasks.EnsureSubnetRoutes(),
```

Comment:
how about a test case for cleanup subnet routes and sync_l2_selfips_and_subnet_routes_flow?

Reply:
Will do in a seperate PR.